include/tvm/relay/attrs/transform.h

```diff
@@ -174,10 +174,11 @@ struct TileAttrs : public tvm::AttrsNode<TileAttrs> {
 
 /*! \brief Attributes used in reverse operators */
 struct ReverseAttrs : public tvm::AttrsNode<ReverseAttrs> {
-  Integer axis;
+  Array<Integer> axes;
```

Comment:
NOTE, as per https://tvm.apache.org/docs/contribute/code_review.html#deliberate-on-api-and-data-structures we shoud always be consistent with numpy API. As a result, let us use the name axis instead. See also https://github.com/apache/incubator-tvm/blob/master/include/tvm/relay/attrs/reduce.h#L36